django/utils/autoreload.py

```diff
@@ -280,8 +280,14 @@ def reloader_thread():
 
 
 def restart_with_reloader():
+    import django.__main__
```

Comment:
If I wrote `import django.__main__` at the top of this file, tests on Jenkis failed: 
https://djangoci.com/job/pull-requests-trusty/database=mysql,label=trusty-pr,python=python3.4/14794/console